pkg/loop/internal/net/client.go

```diff
@@ -52,13 +53,22 @@ func (c *clientConn) Invoke(ctx context.Context, method string, args interface{}
 	cc := c.cc
 	c.mu.RUnlock()
 
+	isClose := method == pb.Service_Close_FullMethodName
+
 	if cc == nil {
+		if isClose {
+			return nil // nothing to do
```

Comment:
Maybe an error should still be returned for these cases? :thinking:

Reply:
Actually, they are arguably different. Below, a terminal error when calling `Close()` on a connected client can probably just be returned as-is. Here, we have no error to return by default, so we would have to construct one. Seems important though, so the caller is not mislead in to thinking the call was successful :thinking: